scripts/gceworker.sh

```diff
@@ -54,6 +55,22 @@ case ${1-} in
     shift
     gcloud compute ssh "${name}" "$@"
     ;;
+    mount)
+    shift
+    if [ $# != 2 ]; then
+      echo "usage: $0 <remote-path> <local-path>"
+      exit 1
+    fi
+    if ! hash sshfs 2>/dev/null; then
+      echo "sshfs not found (install sshfs)"
```

Comment:
Can these be more specific? OSX `brew install sshfs` Linux `sudo apt-get install sshfs`.